src/features/views/store.ts

```diff
@@ -152,12 +152,10 @@ const viewsSlice = createSlice({
       }
     },
     columnAdded: (state, action: PayloadAction<[number, ZetkinViewColumn]>) => {
-      const [viewId, column] = action.payload;
+      const [viewId] = action.payload;
       const colList = state.columnsByViewId[viewId];
       if (colList) {
-        colList.items = colList.items.concat([
-          remoteItem(column.id, { data: column }),
-        ]);
+        colList.isStale = true;
```

Comment:
Why is this necessary? Adding a column was working before this PR, right? So why not keep the old logic for just concatenating it to the end of the list of columns?

Reply:
I tried changing back to the previous code and it still worked fine. So if you can please explain why you found this change to be necessary I'd appreciate that.